### DISCUSSION TITLE
Does this work when the Exchange server disables IMAP/SMTP?

### DISCUSSION THREAD

**Discussion by raj-magesh** [simonrob/email-oauth2-proxy#308] [Q&A] [answered]
Unfortunately, my university disables Outlook IMAP/SMTP access, so I've been using Davmail, but have been having issues recently. Would this project work when IMAP/SMTP is disabled on the server side?

Judging from the last README section on alternatives, I guess not, but I thought it'd be worth asking.

**Comment by simonrob** [comment, marked as answer]
I'm afraid not – the proxy only works if IMAP, POP or SMTP are enabled. [DavMail](https://davmail.sourceforge.net/) is your best bet otherwise, though if it's just SMTP you're after, it may be worth trying [SMTP2Graph](https://www.smtp2graph.com/) or similar alternatives.

**Reply by raj-magesh** [reply]
Thanks!
